script/start-prover-relayer.sh

```diff
@@ -87,8 +92,10 @@ if [ "$ENABLE_PROVER" = "true" ]; then
         ARGS="${ARGS} --tx.sendTimeout ${TX_SEND_TIMEOUT}"
     fi
 
-    exec taiko-client prover ${ARGS}
-else
-    echo "PROVER IS DISABLED"
-    sleep infinity
-fi
+    # Run the prover with error handling
+    if [ "$ENABLE_PROVER" = "true" ]; then
```

Comment:
@richardgreg this condition is already checked here in [L11](https://github.com/taikoxyz/simple-taiko-node/blob/11e77ea27878aa388c173bbe1b06d434c4cc5180/script/start-prover-relayer.sh#L11), also i believe you're missing an `fi` at the end of the script

Reply:
Also, could refactor the handle_error() function into a util script and just `source` it into the start-prover-relayer and start-proposer scripts :)